index.html

```diff
@@ -184,6 +184,27 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
       </div>
     </section>
   </div>
+  <section class="contact-section">
+    <div class="contact-content">
+      <h2 class="title contact-text contact-title">Contact me</h2>
+      <p class="contact-text">If you have an application you are interested in developing, a feature that you need built
+        or a project that needs coding. I’d love to help with it </p>
+      <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/mnqlrawl"
```

Comment:
The form should have full page width but the project has a form with more space from the right. KIndly see the screenshot below and design the form according to the original design
![form](https://user-images.githubusercontent.com/35504149/135316646-85960a30-35cf-4c47-8c16-ebeee063deb1.png)
.